sys-apps/ignition/files/0007-provider-qemu-apply-fw_cfg-patch.patch

```diff
@@ -31,7 +31,7 @@ index a4ebec6a..0ae9cd66 100644
 -	firmwareConfigSizePath = "/sys/firmware/qemu_fw_cfg/by_name/opt/com.coreos/config/size"
 +var (
 +	firmwareConfigPaths = map[string]string{
-+		"/sys/firmware/qemu_fw_cfg/by_name/opt/org.flatcar-linux/config/raw": "/sys/firmware/qemu_fw_cfg/by_name/opt/org.flatcar-linux/config/size",
++		"/sys/firmware/qemu_fw_cfg/by_name/opt/org.flatcar/config/raw": "/sys/firmware/qemu_fw_cfg/by_name/opt/org.flatcar/config/size",
```

Comment:
This should stay `org.flatcar-linux/config/raw` or we introduce a breaking change

Reply:
Ouh thanks, I indeed fixed the same on Mantle earlier.